src/commands.ts

```diff
@@ -26,6 +26,9 @@ function helpCommand(ctx: Context) {
 function clearCommand(ctx: Context) {
   if (!ctx.session.admin) return;
   db.closeAll();
+  cache.ticketIDs.length = 0;
```

Comment:
Right, but it matters after the first message. 
After the first message, user receives a notification about the opening of the ticket. If the ticket was closed and the user has a new question, a new ticket will actually be opened, but the user will not receive notification about this due to the cache. https://github.com/bostrot/telegram-support-bot/blob/42eb04bd23ea5857b9402f90ffe35109a8e30edc/src/users.ts#L88

Reply:
I see. Wouldn't it make more sense just to set it to empty again instead of setting the length?

    ticketIDs = []
    ticketStatus = {}
    ticketSent = []

Reply:
These are equivalent solutions ;)

Reply:
I never realized that there actually is a difference (according to https://stackoverflow.com/a/4804332). I just thought of initializing objects again as a better way (in the sense that is easier to read coming from other languages). But I guess in the end with the JS garbage collector it probably doesn't make a difference.